app/forms/curation_concerns/forms/collection_edit_form.rb

```diff
@@ -12,6 +12,31 @@ class CollectionEditForm
       def required?(key)
         model_class.validators_on(key).any? { |v| v.is_a? ActiveModel::Validations::PresenceValidator }
       end
+
+      # @return [Hash] All generic files in the collection, file.to_s is the key, file.id is the value
+      def select_files
```

Comment:
What does this have to do with using to_s?  Should this be moved to a separate PR?